roadservice/core/models.py

```diff
@@ -233,13 +293,89 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.type = Role.Type.COUNTY_MODERATOR
 
-    # TODO
-    def assign_county_expert(self):
-        pass
+    @property
+    def county(self):
+        return self.region.get_concrete()
 
     def get_concrete(self):
         return self
 
+    def add_service_team(self, speciality, members_users):
+        for user in members_users:
+            if user.has_role():
+                return None
+        new_team = ServiceTeam.objects.create(county=self.county, speciality=speciality)
+        for user in members_users:
+            Serviceman.objects.create(team=new_team, user=user)
+        return new_team
+
+    def edit_service_team(self, team, speciality, members_users):
+        ex_members_users = [serviceman.user for serviceman in team.members.all()]
+        for user in members_users:
+            if user.has_role() and user not in ex_members_users:
+                return False
+        for user in members_users:
+            if not user.has_role():
+                Serviceman.objects.create(user=user, team=team)
+        for serviceman in team.members.all():
+            if serviceman.user not in members_users:
+                serviceman.delete()
+        team.speciality = speciality
+        team.save()
+        return True
+
+    def delete_service_team(self, team):
+        for serviceman in team.members.all():
+            serviceman.delete()
+        team.delete()
```

Comment:
As you remember, we don't delete teams. We mark them as deleted. Why? It had something to do with reporting. I don't remember.